pl/tutorials/tour/implicit-conversions.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: tutorial
+title: Konwersje implicit
+
+disqus: true
+
+tutorial: scala-tour
+num: 26
+language: pl
+tutorial-next: polymorphic-methods
+tutorial-previous: implicit-parameters
+---
+
+Konwersja implicit z typu `S` do `T` jest określona przez wartość implicit, która jest funkcją typu `S => T` lub przez metodę implicit odpowiadającą funkcji tego typu.
```

Comment:
I thought about this, but `niejawne` in Polish suggests something that is just opposite of explicit. This might work for implicit conversions, but for implicit parameters we should probably use `domniemany` since implicit here means more like something that is implied.